<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="32.873" tests="1" failures="1">
  <testsuite name="Root Suite" timestamp="2025-06-17T18:15:38" tests="0" file="cypress\e2e\tests\TransactionTest.cy.js" time="0.000" failures="0">
  </testsuite>
  <testsuite name="Validating Transaction update" timestamp="2025-06-17T18:15:38" tests="1" time="32.862" failures="1">
    <testcase name="Validating Transaction update should be able to add transaction to the account" time="14.750" classname="should be able to add transaction to the account">
      <failure message="Timed out retrying after 5000ms: expected 6422 to not equal 6422" type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 5000ms: expected 6422 to not equal 6422
    at Context.eval (webpack://cypress-framework/./cypress/e2e/pages/BudgetPage.js:53:29)

      + expected - actual

]]></failure>
    </testcase>
  </testsuite>
</testsuites>